ui/app/templates/components/selectable-card-container.hbs

```diff
@@ -0,0 +1,74 @@
+{{#linked-block
+  "vault.cluster.metrics.http-requests"
+  class="grid-item-http"
+}}
+  <SelectableCard 
+    @cardTitle="Http Requests" 
```

Comment:
you might've already answered this and i could've missed it, but should this be `HTTP Requests` or `HTTP Request Volume` to match the title on the following page?